### PR TITLE
fix: Update Tooltips

### PR DESCRIPTION
**Pull request by Xhy-5000**
Fix: https://github.com/casdoor/casdoor/issues/1309

### CODE REVIEW

**Comment by hsluoyz** [comment]
@imchell plz review

File: web/src/locales/en/data.json [hunk 1]
```diff
@@ -10,64 +10,64 @@
...
     "Sync policies successfully": "Sync policies successfully"
   },
   "application": {
     "Always": "Always",
     "Auto signin": "Auto signin",
-    "Auto signin - Tooltip": "Auto signin - Tooltip",
+    "Auto signin - Tooltip": "If a login session exists in Casdoor, the system automatically uses the session to log in to the application",
     "Background URL": "Background URL",
-    "Background URL - Tooltip": "Background URL - Tooltip",
+    "Background URL - Tooltip": "Login page background image URL",
```
Comment (imchell) [inline web/src/locales/en/data.json:23]: It is not only the login page background (but also sign up page, etc.)

File: web/src/locales/en/data.json [hunk 2]
```diff
@@ -161,94 +161,94 @@
...
     "Phone prefix": "Phone prefix",
-    "Phone prefix - Tooltip": "Phone prefix - Tooltip",
+    "Phone prefix - Tooltip": "Phone prefix",
     "Preview": "Preview",
-    "Preview - Tooltip": "Preview - Tooltip",
+    "Preview - Tooltip": "Preview",
     "Products": "Products",
     "Provider": "Provider",
-    "Provider - Tooltip": "Provider - Tooltip",
+    "Provider - Tooltip": "Provider that need to be configured for third-party login",
```
Comment (imchell) [inline web/src/locales/en/data.json:228]: Grammatical mistake

File: web/src/locales/en/data.json [hunk 3]
```diff
@@ -161,94 +161,94 @@
...
+    "Signup URL - Tooltip": "Signup URL",
     "Signup application": "Signup application",
-    "Signup application - Tooltip": "Signup application - Tooltip",
+    "Signup application - Tooltip": "Indicates which application the user registered through when registering",
     "Sorry, the page you visited does not exist.": "Sorry, the page you visited does not exist.",
     "Sorry, the user you visited does not exist or you are not authorized to access this user.": "Sorry, the user you visited does not exist or you are not authorized to access this user.",
     "Sorry, you do not have permission to access this page or logged in status invalid.": "Sorry, you do not have permission to access this page or logged in status invalid.",
     "State": "State",
-    "State - Tooltip": "State - Tooltip",
+    "State - Tooltip": "Statep",
```
Comment (imchell) [inline web/src/locales/en/data.json:251]: Not sure what Statep means, is that a typo?

File: web/src/locales/en/data.json [hunk 4]
```diff
@@ -274,13 +274,13 @@
...
     "Address": "Address",
     "Admin": "Admin",
-    "Admin - Tooltip": "Admin - Tooltip",
+    "Admin - Tooltip": "Admin",
     "Admin Password": "Admin Password",
-    "Admin Password - Tooltip": "Admin Password - Tooltip",
+    "Admin Password - Tooltip": "Admin Password",
     "Auto Sync": "Auto Sync",
-    "Auto Sync - Tooltip": "Auto Sync - Tooltip",
+    "Auto Sync - Tooltip": "Automatic synchronization configuration, disabled when 0",
```
Comment (imchell) [inline web/src/locales/en/data.json:281]: synchronization configuration -> configuration synchronization

File: web/src/locales/en/data.json [hunk 5]
```diff
@@ -10,64 +10,64 @@
     "Failed to sync policies": "Failed to sync policies",
```
Comment (hsluoyz) [inline web/src/locales/en/data.json:10]: Update zh too

**Review by hsluoyz — CHANGES_REQUESTED** [review]

**Comment by hsluoyz** [comment]
@Xhy-5000 fix conflicts

image

**Comment by hsluoyz** [comment]
Replaced by: https://github.com/casdoor/casdoor/pull/1475

**Pull request closed without merging**